wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/Sections/GroupOptionsSectionController.swift

```diff
@@ -81,7 +81,7 @@ final class GroupOptionsSectionController: GroupDetailsSectionController {
     // MARK: - Collection View
 
     override var sectionTitle: String {
-        return "participants.section.settings".localized(uppercased: true)
+        return L10n.Localizable.Participants.Section.settings.localizedUppercase
```

Comment:
By the way I had a talk with Astrid yesterday and she said that this "uppercase" does not work in all languages. So it would be better to let the copywriters handle it manually.
No need to change here now, but as a general information 😄

Reply:
Thank you for the heads up. I will gather all appropriate information and I will handle it on a separate PR.